rolldown-contract/src/rolldown.sol

```diff
@@ -279,6 +284,30 @@ contract RollDown {
         }
     }
 
+     function process_pending_update_deposit(
```

Comment:
this is l1. and no, the deposit was processed, unsuccessfully, tokens are returned to user since none were minted on l2. We cannot reengage the same request id again, because it will break the counters. something went wrong, try another deposit